tests/integration/BUILD

```diff
@@ -1,10 +1,9 @@
-package(default_visibility=["//visibility:public"])
```

Comment:
Entries for `//tests/...` can easily be excluded if you'd prefer they not exist, but they are technically importable by third parties since `bzl` files come with an implicit public visibility in the context of Bazel's `load` statements unless they live in a directory called `private`.